src/Background.cpp

```diff
@@ -39,7 +39,7 @@ static ThemeMetric<bool> RAND_BG_ENDS_AT_LAST_BEAT("Background", "RandomBGEndsAt
 
 static Preference<bool>	g_bShowDanger( "ShowDanger", true );
 static Preference<float> g_fBGBrightness( "BGBrightness", 0.7f );
-static Preference<RandomBackgroundMode> g_RandomBackgroundMode( "RandomBackgroundMode",	BGMODE_RANDOMMOVIES );
+static Preference<RandomBackgroundMode> g_RandomBackgroundMode( "RandomBackgroundMode",	BGMODE_OFF );
```

Comment:
I think we can leave this as BGMODE_RANDOMMOVIES. This would only get applied if people place RandomMovies in the folder which by default is empty and it seems nicer that if someone puts movies here then they should take effect immediately.

Reply:
good point!